### DISCUSSION TITLE
POD discussion: Wheeler-Kiladis spectra

### DISCUSSION THREAD

**Discussion by tsjackson-noaa** [NOAA-GFDL/MDTF-diagnostics#123] [Q&A]
Thread for maintenance and updates to the Wheeler-Kiladis POD.

**Comment by tsjackson-noaa** [comment]
Description of pull request NOAA-GFDL/MDTF-diagnostics#45 / NOAA-GFDL/MDTF-diagnostics#46 from @bitterbark :

The Wheeler-Kiladis code now can reproduce the obs_data'figures with provided NCEP reanalysis data. It can also better handle different observations with more flexibility in time variable/calendar descriptions, and by crudely handling missing values. Note that the latter does not produce great results. This commit is accompanied by an updated obs_data.tar with the new obs figures that show the same data as before but are formatted exactly like the model figures. See figures )

**Comment by tsjackson-noaa** [comment]
The branch containing existing development work on this POD has been renamed from pod/Wheeler_Kiladis to [feature/update_Wheeler_Kiladis](https://github.com/NOAA-GFDL/MDTF-diagnostics/tree/feature/update_Wheeler_Kiladis), as part of the reorganization proposed in issue NOAA-GFDL/MDTF-diagnostics#106. **Note** that this branch contains changes from PR NOAA-GFDL/MDTF-diagnostics#46 but not those from PR NOAA-GFDL/MDTF-diagnostics#69, and will be deleted when the latter PR is closed.

**Reply by ahmedfiaz** [reply]
@bitterbark @tsjackson-noaa 
I'm testing this POD on CMIP6 data. This POD requires `ua200` as required input whereas most models with daily CMIP6 output only have `ua250` available. I suggest including `ua250` as an alternate.

**Comment by bitterbark** [comment]
Noted, thanks!
